main.tf

```
@@ -0,0 +1,279 @@
+locals {
+  enabled                      = module.this.enabled
+  network_firewall_name        = var.network_firewall_name != null && var.network_firewall_name != "" ? var.network_firewall_name : module.this.id
```

can we use this instead?

```suggestion
  network_firewall_name        = try(length(var.network_firewall_name), 0) > 0 ? var.network_firewall_name : module.this.id
```